core_pb/src/messages/mod.rs

```diff
@@ -1,4 +1,5 @@
 #[cfg(feature = "std")]
+use crate::grid::standard_grid::StandardGrid;
```

Comment:
this should probably be above the std cfg because the cfg is supposed to apply to serverstatus

Reply:
good catch